app/code/Codifi/CustomerRequest/Setup/Patch/Data/AddRequestSendCmsBlock.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\CustomerRequest\Setup\Patch\Data;
+
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\Cms\Model\BlockFactory;
+use Magento\Cms\Model\BlockRepository;
+
+/**
+ * Class AddRequestSendCmsBlock
+ * @package Codifi\CustomerRequest\Setup\Patch\Data
+ */
+class AddRequestSendCmsBlock implements DataPatchInterface
```

Comment:
Actually, id of the block is 'send-request', so it's better to name data patch AddSendRequestCmsBlock